stdlib/public/Platform/CMakeLists.txt

```diff
@@ -6,28 +6,21 @@ set(swift_platform_sources
     TiocConstants.swift
     tgmath.swift.gyb)
 
-if(SWIFT_HOST_VARIANT MATCHES "${SWIFT_DARWIN_VARIANTS}")
-  set(swift_platform_name swiftDarwin)
-  set(swift_platform_sources
-      Darwin.swift
-      ${swift_platform_sources}
-      POSIXError.swift
-      MachError.swift)
-  set(swift_platform_flags
-      SWIFT_COMPILE_FLAGS -Xfrontend -disable-objc-attr-requires-foundation-module
-      API_NOTES_NON_OVERLAY)
-else()
-  set(swift_platform_name swiftGlibc)
-  set(swift_platform_sources
-      Glibc.swift
-      ${swift_platform_sources})
-  set(swift_platform_flags
-      DEPENDS glibc_modulemap)
-endif()
+add_swift_library(swiftDarwin ${SWIFT_STDLIB_LIBRARY_BUILD_TYPES} IS_SDK_OVERLAY
+    Darwin.swift
+    ${swift_platform_sources}
+    POSIXError.swift
+    MachError.swift
+
+    SWIFT_COMPILE_FLAGS -Xfrontend -disable-objc-attr-requires-foundation-module
+    TARGET_SDKS IOS;OSX;TVOS;WATCHOS
```

Comment:
The syntax of `TARGET_SDKS` does not require semicolons, see, for example, `stdlib/public/SDK/CloudKit/CMakeLists.txt`.